src/core/include/units/quantity.h

```diff
@@ -139,7 +139,9 @@ class quantity {
 
   template<typename Value>
     requires(safe_convertible_to_<std::remove_cvref_t<Value>, rep>)
-  constexpr explicit(!detail::quantity_one<quantity>) quantity(Value&& v) : number_(std::forward<Value>(v))
+  constexpr explicit(!detail::quantity_one<quantity>)
+    quantity(Value&& v) noexcept(std::is_nothrow_constructible_v<Rep, Value&&>) :
```

Comment:
Either we add `noexcept` to all similar member functions in all quantity-like types, or I would prefer not to add this one case.

Reply:
Can do, and will do if there is a chance to be accepted.

Reply:
Please add the remaining `noexcept` specifiers according to the LEWG policy, and I will be happy to accept those.